ya_httpx_client/service/service_base.py

```diff
@@ -0,0 +1,35 @@
+from abc import ABC
+
+from yapapi.services import Service
+
+
+class AbstractServiceBase(ABC, Service):
+    '''Base class for all services. Contains common things, inheriting classes
+    are expected to implemented `run` method.'''
+    PROVIDER_URL = ''  # TODO this will be removed with https://github.com/golemfactory/ya-httpx-client/issues/14
+
+    def __init__(self, *args, start_steps, request_queue, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.start_steps = start_steps
+        self.queue = request_queue
```

Comment:
Since it's a base class type hints here would be useful I think.

Reply:
I will add them in the next PR (#15 ) because `start_steps` disappears there.

Reply:
145fd39